src/Keboola/GoodDataWriter/App.php

```diff
@@ -34,10 +35,36 @@ public function __construct($consoleOutput)
         $this->temp->initRunFolder();
     }
 
+    public function checkProjectAccess(Client $gdClient, ProvisioningClient $provisioningClient, array $config)
+    {
+        try {
+            $gdClient->get("/gdc/projects/{$config['parameters']['project']['pid']}");
+        } catch (Exception $e) {
+            if ($e->getCode() !== 403) {
+                throw $e;
+            }
+
+            if (!getenv('KBC_TOKEN')) {
+                throw new \Exception('KBC Token is missing from the environment');
+            }
+            $provisioningClient->addUserToProject(
```

Comment:
tu by mohlo zalogovat nieco ako ze `do projektu pridava uzivatela pre load dat..` ci?

Reply:
přidal jsem to semka: https://github.com/keboola/gooddata-writer-v3/pull/6/files#diff-58980ac23a3a9a0b92abd9805eed1e78R48